### ISSUE TITLE
Creality Slicer Sourcecode

### ISSUE THREAD

**Issue by sdaitzman** [Creality3DPrinting/Ender-3#55]
Hello! Is the Creality Slicer sourcecode available? I would love to refer to it.

**Comment by cyber-murmel** [comment]
I think there is not "the one slicer".
You can use cura for the Ender 3.
https://github.com/Ultimaker/Cura

**Comment by sdaitzman** [comment]
Yep, I also primarily use Cura, but Creality has their own slicer: https://www.creality.com/download/slicing-software_c0010

I suspect it's based on Cura or Slic3r, but could be wrong. Creality Slicer's rafts are much nicer, and it's only available as an msi Windows installer. I'd love to either port its raft to native Cura, or recompile it for macOS/Linux so I can use it without virtualization.

**Comment by cyber-murmel** [comment]
Ah, good to know.

**Comment by Joe-Bourne** [comment]
hey @sdaitzman, you're correct - when you load the Creality slicer, look at the bottom right of the splash screen or the Help->About menu and you will see the words "Powered by Cura"
